examples/switch/css/switch-checkbox.css

```diff
@@ -18,17 +18,15 @@ label {
 label .label {
   display: inline-block;
   width: 9em;
-  -webkit-user-select: none;
```

Comment:
@jongund I think you added these specifically before, but the updated Stylelint PR cleaned out a bunch. If you think it's needed they can be ignored https://stylelint.io/user-guide/rules/list/property-no-vendor-prefix/ or Autoprefixer could be added to the build to add any needed ones based on the supported browsers